keepassxc-browser/background/event.js

```diff
@@ -178,6 +178,25 @@ kpxcEvent.onHTTPAuthPopup = function(tab, data) {
     browserAction.stackUnshift(stackData, tab.id);
     page.tabs[tab.id].loginList = data;
     browserAction.show(tab);
+
+    const dialogData = {
+        width: 480, height: 250,
```

Comment:
Have you checked this with HiDPI?

Reply:
How can I check this without having an HiDPI screen?
Does multiplying the size using window.devicePixelRatio helps?

Reply:
Just checked and popup.css has max-width, so it will not change if we double the window size for HiDPI, so I'll just keep that...